From 8661c383ec80563a1f6434b25654a5e968ed18c2 Mon Sep 17 00:00:00 2001
From: James M. Willenbring <jmwille@sandia.gov>
Date: Thu, 29 Oct 2015 09:04:39 -0600
Subject: [PATCH] Removing FEApp from the packages list.

---
 PackagesList.cmake |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/PackagesList.cmake b/PackagesList.cmake
index b22bf85..4752f80 100644
--- a/PackagesList.cmake
+++ b/PackagesList.cmake
@@ -131,7 +131,6 @@ TRIBITS_REPOSITORY_DEFINE_PACKAGES(
   MeshingGenie          packages/meshinggenie             EX
   TrilinosCouplings     packages/trilinoscouplings        SS
   Pike                  packages/pike                     SS
-  FEApp                 demos/FEApp                       SS # Capability demonstration package
   )
 
 # Allow builds even if some packages are missing
-- 
1.7.1

