From 2b74c91fb9c712a1b3eaab4e30b2b95c9c1d36a9 Mon Sep 17 00:00:00 2001
From: Brent Perschbacher <bmpersc@sandia.gov>
Date: Thu, 29 Oct 2015 08:25:49 -0600
Subject: [PATCH 6/9] Remove missing RBGen from Tribits example.

---
 cmake/tribits/examples/MockTrilinos/PackagesList.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmake/tribits/examples/MockTrilinos/PackagesList.cmake b/cmake/tribits/examples/MockTrilinos/PackagesList.cmake
index 9f43709..77186a1 100644
--- a/cmake/tribits/examples/MockTrilinos/PackagesList.cmake
+++ b/cmake/tribits/examples/MockTrilinos/PackagesList.cmake
@@ -20,7 +20,6 @@ TRIBITS_REPOSITORY_DEFINE_PACKAGES(
   ML                  packages/ml                     PT
   Belos               packages/belos                  ST
   Stratimikos         packages/stratimikos            PT
-  RBGen               packages/rbgen                  PT
   Phalanx             packages/phalanx                ST
   Panzer              packages/panzer                 ST
   AlwaysMissing       AlwaysMissing                   PT
-- 
2.1.3

